test/e2e_closure_test.ts

```diff
@@ -49,7 +49,6 @@ describe('golden file tests', () => {
         'checkRegExp',
         'checkTypes',
         'checkVars',
-        'commonJsModuleLoad',
```

Comment:
One reason I keep harping about separate CLs is that otherwise we have no context on why changes like this were made, except that "Martin noticed it while changing 3000 other lines of code".  And you could explain each of these by "it was necessary", but it'd be nice to have a log of why.  For this particular one, given that we're trying to preserve existing outputs, why is it needed?